lib/internal/errors.js

```diff
@@ -728,6 +728,9 @@ E('ERR_HTTP2_STREAM_SELF_DEPENDENCY',
 E('ERR_HTTP2_UNSUPPORTED_PROTOCOL', 'protocol "%s" is unsupported.', Error);
 E('ERR_HTTP_HEADERS_SENT',
   'Cannot %s headers after they are sent to the client', Error);
+E('ERR_HTTP_INCOMING_SOCKET_ENCODING',
+  'Incoming socket encoding is not allowed (RFC 2616)',
```

Comment:
Can you please change this to e.g., `Changing the incoming socket encoding is not possible (see RFC 2616)`?.